_translator-files/po/sv-SE/Client-Troubleshooting.po

```diff
@@ -70,276 +71,278 @@ msgid ""
-msgstr ""
+msgstr "Du kan ställa in din \"New Client Level\" till ett lågt värde (t.ex. 10), eller ställa in musikerna du spelar med till \"Solo\"-läge (i mixerpanelen på höger sida). På så sätt blir antingen nya deltagare väldigt tysta, eller så hör du dem inte alls."
 
 #. type: Title ###
 #: ../wiki/en/Client-Troubleshooting.md:71
 #, markdown-text, no-wrap
 msgid "Can't see the Server you want to join?"
-msgstr ""
+msgstr "Kan du inte se servern du vill gå med i?"
 
 #. type: Plain text
 #: ../wiki/en/Client-Troubleshooting.md:74
 #, markdown-text
 msgid "First check that you have the right genre Server selected in your Connect window. Sometimes network issues mean your Client won't list all the available Servers. If you know the name of the Server you want to join, you can [look up its IP address here](https://explorer.jamulus.io/). Enter that address in the \"Server Address\" field in the Connect Setup window to connect to it."
-msgstr ""
+msgstr "Kontrollera först att du har valt rätt genre Server i ditt Connect-fönster. Ibland innebär nätverksproblem att din klient inte kommer att lista alla tillgängliga servrar. Om du vet namnet på servern du vill gå med i kan du [söka upp dess IP-adress här](https://explorer.jamulus.io/). Ange den adressen i fältet \"Serveradress\" i fönstret Connect Setup för att ansluta till den."
 
 #. type: Title ###
 #: ../wiki/en/Client-Troubleshooting.md:75
 #, markdown-text, no-wrap
 msgid "Not seeing a list of Servers at all?"
-msgstr ""
+msgstr "Ser du inte en lista över servrar alls?"
 
 #. type: Plain text
 #: ../wiki/en/Client-Troubleshooting.md:78
 #, markdown-text
 msgid "In the UK (and possibly other regions/routers) the Virgin Media Cable Internet Modem setting can cause an issue. \"Block Fragmented IP Packets\" should not be checked. For other routers/ISP, also try turning off SPI (Stateful Packet Inspection) and if that fixes it, you can make a judgment about whether to leave that off or not."
-msgstr ""
+msgstr "I Storbritannien (och möjligen andra regioner/routrar) kan inställningen av Virgin Media Cable Internet Modem orsaka problem. \"Blockera fragmenterade IP-paket\" ska inte markeras. För andra routrar/ISP, prova också att stänga av SPI (Stateful Packet Inspection) och om det fixar det kan du göra en bedömning om du ska lämna det av eller inte."
 
 #. type: Plain text
 #: ../wiki/en/Client-Troubleshooting.md:80
 #, markdown-text
 msgid "In some cases, it may be your ISP that is blocking your use of Jamulus. See the note on the [server troubleshooting page](Server-Troubleshooting#nobody-can-connect-to-my-server---but-i-can-connect-locally) about \"Nobody can connect to my Server\""
-msgstr ""
+msgstr "I vissa fall kan det vara din internetleverantör som blockerar din användning av Jamulus. Se anteckningen på [serverfelsökningssidan](Server-felsökning#ingen-kan-ansluta-till-min-server---men-jag-kan-ansluta-lokalt) om \"Ingen kan ansluta till min server\""
```

Comment:
"Server-Troubleshooting" in link shouldn't be translated.

Reply:
Crucial to get right.